net-dialup/cutecom/cutecom-0.22.0.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Graphical serial terminal"
+DESCRIPTION="CuteCom is a graphical serial terminal, like minicom."
+HOMEPAGE="http://cutecom.sourceforge.net"
+COPYRIGHT="2004-2009 Alexander Neundorf"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://cutecom.sourceforge.net/cutecom-$portVersion.tar.gz"
+CHECKSUM_SHA256="1b6620a6159cf3d50bb36cce544e91486817df7f1d553bf239d6db6108dd2ea5"
+PATCHES="cutecom-$portVersion.patchset"
+ADDITIONAL_FILES="cutecom.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cutecom$secondaryArchSuffix = $portVersion
+	app:cutecom$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQtCore$secondaryArchSuffix
+	lib:libQtGui$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQtCore$secondaryArchSuffix
+	devel:libQtGui$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+PATCH()
+{
+	sed -i \
+		-e "s|share/man\(/man1\)|$relativeManDir\1|" \
+		-e "/binInstallDir bin/ s|bin|$relativeAppsDir|2" \
+		CMakeLists.txt
+}
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix ..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	mv $appsDir/cutecom $appsDir/CuteCom
```

Comment:
I would drop the mv (i.e. keep the runtime all lowercase) and only use CuteCom for the AppDeskbarSymlink.
On GNU/Linux, for instance, the runtime is called cutecom, so I think we should stick to that (and only use CuteCom on the DeskBar.)